framework/src/wifi_manager/wifi_manager_lwnl.c

```diff
@@ -45,6 +45,30 @@ typedef enum {
 	WIFI_UTILS_LWNL_WRITE_FAILED,
 } wu_lwnl_status_e;
 
+static inline wifi_utils_result_e _convert_trwifi_result(trwifi_result_e res)
+{
+	switch (res) {
+	case WIFI_UTILS_FAIL:
```

Comment:
There is no `WIFI_UTILS_FAIL` in `trwifi_result_e`, but you checked the `res` with this.
Is it okay?